packages/astro/src/core/build/index.ts

```diff
@@ -40,7 +40,9 @@ export interface BuildOptions {
 	 * Teardown the compiler WASM instance after build. This can improve performance when
 	 * building once, but may cause a performance hit if building multiple times in a row.
 	 *
-	 * @internal only used for testing
+	 * When building multiple projects in the same execution (e.g. during tests), disabling
+	 * this option can greatly improve performance at the cost of some extra memory usage.
+	 *
```

Comment:
I'd prefer to keep this internal still. I imagine for external libraries this shouldn't have a significant impact if not used since it only mattered for Astro having tons of tests.

Reply:
Astro has around 2,000 tests. For companies requiring high reliability, it is not uncommon to have very exhaustive testing on libraries (at least for the companies I know that fit that description, including mine).

Multiple of our libraries for Next have over 10k tests each. Libraries for Astro wouldn't be any less tested.

Reply:
This isn't related to the number of test, but the number of builds for tests. Astro has many builds to test different permutations of all features and I doubt the same is needed for external libraries. Even so, Astro's test suite is due for some fixtures reduction because some fixtures could be combined, so in practice this option isn't really encouraged.